src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -101,6 +101,19 @@
      */
     private SctpManager sctpManager;
 
+    /**
+     * The time at which this endpoint was created (in millis since epoch)
+     */
+    //TODO: use clock/instant in this file
+    private final Long creationTimeMillis;
+
+    /**
+     * How long we'll give an endpoint to either successfully establish
+     * an ICE connection or fail before we expire it.
+     */
+    //TODO: make this configurable
+    private static final Duration EP_TIMEOUT = Duration.ofMinutes(2);
```

Comment:
this was somewhat arbitrary...seemed reasonable but i'd be fine with changing it.

Reply:
I agree it's reasonable.  (If anything it's probably over-generous.)